### ISSUE TITLE
cannot find symbol error on build

### ISSUE THREAD

**Issue by Reprevise** [ponnamkarthik/FlutterToast#195]
When I try and build my app, I get this error.

```
C:\dev\flutter_apps\client_apps\arrow_clothing_app\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java:21: error: cannot find symbol
    flutterEngine.getPlugins().add(new io.github.ponnamkarthik.toast.fluttertoast.FluttertoastPlugin());
                                                                                 ^
  symbol:   class FluttertoastPlugin
  location: package io.github.ponnamkarthik.toast.fluttertoast
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 19s
Exception: Gradle task assembleDebug failed with exit code 1
```

I'm not using `Fluttertoast.showToast()`, I'm using the new code via the `FToast` class.

**Comment by chris-pro** [comment]
Same problem using **Fluttertoast.showToast()**

`
D:\...\myapp\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java:20: error: cannot find symbol
    flutterEngine.getPlugins().add(new io.github.ponnamkarthik.toast.fluttertoast.FluttertoastPlugin());
                                                                                 ^
  symbol:   class FluttertoastPlugin
  location: package io.github.ponnamkarthik.toast.fluttertoast
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 41s
Running Gradle task 'assembleRelease'...                                
Running Gradle task 'assembleRelease'... Done                      42,9s
Gradle task assembleRelease failed with exit code 1
`

I also cannot launch my app in debug mode

**Comment by ponnamkarthik** [comment]
@Reprevise @chris-pro Fixed in `7.0.1`
